### PR TITLE
hehehehehehhehrehhthe

### PR DESCRIPTION
**Pull request by gageway1**
inb4 magic numbers
overruled

### CODE REVIEW

**Comment by gageway1** [comment]
don't merge until main is merged and deployed

File: helpers/messages.py [hunk 1]
```diff
@@ -88,8 +88,14 @@ async def run_message_redaction(message: discord.Message):
...
-        except Exception as e:
-            logging.error(f"Error while reacting to message: {e}")
\ No newline at end of file
+    if "http" not in message.content.lower():
+        if "js" in message.content.lower():
+            try:
+                await react_with_funny_letters(message, JSBAD)
+            except Exception as e:
+                logging.error(f"Error while reacting to message: {e}")
+        if message.author.id == 284876500480557056 and random.randint(1, 50) == 1: # pent's ID because funnee
```
Comment (isthistechsupport) [inline helpers/messages.py:97]: I specifically approve of this idea (we should add as many annoying emojis as we can) (some epilepsy-inducing animated emoji would also work)

**Review by isthistechsupport — APPROVED** [review]
wholeheartedly approved

**Review by isthistechsupport — APPROVED** [review]
LGTM

**Pull request merged**